text/66-self-managed-stack-sets.md

```diff
@@ -0,0 +1,329 @@
+  executionRoleName: 'AWSCloudFormationStackSetExecutionRole',
+  faultTolerancePercentage: 10,
+  maxConcurrentPercentage: 10,
+  regionConcurrencyType: cdk.StackSetRegionConcurrency.PARALLEL,
+  //...
+});
+```
+
+**Deploy Stack Set:**
+
+To deploy the stack set, the CLI command remains the same: `cdk deploy poc-stack-set`.
+If the stack set **does not exist**, then CDK will create the stack set directly in CloudFormation.
+If the stack set **does exist**,
+then the stack set itself will be updated as well as all **existing** stack set instances.
+
+**Creating Stack Set Instances:**
+
+Before creating stack set instances, child accounts should be bootstrapped (or custom stack set roles created) and the initial stack set deployed.
+Stack set instances can be added to the stack set by console/cli/api by specifying the child account id and region.
+Future stack set deployments though CDK will update the stack set itself, and all existing stack set instances.
+
+## FAQ
+
+### What are we launching today?
+
+The core construct StackSet, which will enable users to define stack sets directly from CDK.
+A new Cloud Assembly artifact type for stack sets. CLI changes to enable deploying stack sets through CDK.
+
+### Why should I use this feature?
+
+This core constuct directly creates a CloudFormation stack set vs. provisioning a single stack that deploys a stack set.
+The deployment also updates the stack set, updates existing stack set instances, and monitors the stack set operation.
+
+### What is supported?
+
+* Self Manged Stack Sets
+  * Create, Update, and Update Existing Stack Set Instances directly through CDK
+
+### What is not supported?
+
+* Service Managed Stack Sets
+* Stack Set Instance Add/Remove through CDK
+* Stack Set Deletion
+  * Stack Set deletion will not be included due complexities with collecting/deleting stack set instances before deleting the stack set itself
+* CDK Assets (CDK File Assets/CDK Docker Assets)
+  * CDK Assets will not be include in the iteration due to complexities around asset permissioning accross accounts and regions
+
+## Internal FAQ
+
+### Why are we doing this?
+
+Today, CDK does not support the creation and updating of CloudFormation stack sets directly.
+This solution would also enable deploying to existing stack sets with low effort.
+
+CDK does provide the `cdk.CfnStackSet` construct, which could accomplish similar behavior.
+The difference is that the `cdk.CfnStackSet` construct is a resource in a single CloudFormation stack with the resource type `AWS::CloudFormation::StackSet`,
+where stack provisions the actual stack set.
+This requires a 1:1 mapping of parameters and tags that need to be passed from the stack to the stack set.
+Customer can deploy to existing stack sets, but requires some migration steps.
+The customer would need to create a new stack,
+and import the stack set into the resource `AWS::CloudFormation::StackSet`.
```

Comment:
I'm mostly repeating myself here, but this doesn't cover why this approach is better than using `CfnStackSet`.

We should be talking about this in terms of what we are enabling for users here that didn't previously exist.